_pages/plugins/freq.md

```diff
@@ -0,0 +1,29 @@
+---
+mediawiki: FreQ
+title: FreQ
+description: An ImageJ plugin to analyze pixel intensity oscillations in time-lapse 2D images.
+icon: https://raw.githubusercontent.com/hansenjn/FreQ/master/FreQ%20Logo%20Small.png
+categories: [Automation, Analysis, Cilia, Fast Fourier Transformation, Frequency Analysis]
+doi: 10.1016/j.xpro.2022.101542
+source-url: https://github.com/hansenjn/FreQ
+dev-status: Stable
+support-status: Active
+team-founder: '@hansenjn'
+team-maintainer: '@hansenjn'
+---
+
+FreQ is an ImageJ plugin to analyze pixel intensity oscillations in time-lapse 2D images. 
+For example, this allows to characterize the ciliary beat of multi-ciliated cells in time-lapse images
+acquired with a bright-field or phase-contrast microscope. 
+
+The software is published with the following publication:
+Jeong I, Hansen JN, Wachten D, Jurisch-Yaksi N. 
+*Measurement of ciliary beating and fluid flow in the zebrafish adult telencephalon.*
+STAR Protoc. 2022 Jul 15;3(3):101542. doi: https://doi.org/10.1016/j.xpro.2022.101542.
```

Comment:
Please use the `{% include citatation}` mechanism to cite publications. See https://imagej.net/editing/citations